tools/dawgrun/README.md

```
@@ -50,10 +51,18 @@ To switch the build back to mainline:
 
 ## Running
 
-You are dropped into a prompt:
+Run without arguments to start the REPL:
 
     dawgrun >
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)